mappings/net/minecraft/client/gui/widget/EditBoxWidget.mapping

```diff
@@ -0,0 +1,34 @@
+CLASS net/minecraft/unmapped/C_uqwspvtd net/minecraft/client/gui/widget/EditBoxWidget
+	FIELD f_byyxedyq CURSOR_INSERT_COLOR I
```

Comment:
How about `INSERT_CURSOR`? I'm guessing it's the "box" cursor that appears over a character while in insert mode

Reply:
It's not a box!

There's 2 cursor types: insert and append

Append will display it as an underscore, while insert is a pipe.